storage/kv/merkletreekv/strkv.go

```diff
@@ -0,0 +1,35 @@
+package merkletreekv
+
+import (
+	"bytes"
+
+	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/coniks-sys/coniks-go/storage/kv"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+func StoreSTR(wb kv.Batch, str *merkletree.SignedTreeRoot) error {
+	buff := new(bytes.Buffer)
+	if err := merkletree.EncodeSTR(buff, str); err != nil {
+		return err
+	}
+
+	wb.Put(STRKey(str.Epoch), buff.Bytes())
+	return nil
+}
+
+func LoadSTR(db kv.DB, epoch uint64) (*merkletree.SignedTreeRoot, error) {
+	bbuff, err := db.Get(STRKey(epoch))
+	if err != nil {
+		return nil, err
+	}
+	buff := bytes.NewBuffer(bbuff)
+	return merkletree.DecodeSTR(buff)
+}
+
+func STRKey(epoch uint64) []byte {
```

Comment:
This could be unexported or not? If this should stay exported it would need a more speaking name.